.circleci/config.yml

```diff
@@ -353,17 +353,6 @@ jobs:
           command: |
             cd examples/cra-react15
             yarn storybook --smoke-test --quiet
-  frontpage:
```

Comment:
Please ensure I removed this correctly, as I'm not all that familiar with CircleCI. (Same with TeamCity changes.) 🙏